test/performance-charts/tests/FunnelChart.bench.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+// eslint-disable-next-line no-restricted-imports
+import { render, cleanup } from '@testing-library/react';
+import { bench, describe } from 'vitest';
+import { FunnelChart } from '@mui/x-charts-pro/FunnelChart';
+import { LicenseInfo, generateLicense } from '@mui/x-license';
+import { options } from '../utils/options';
+
+describe('FunnelChart', () => {
+  const dataLength = 100;
```

Comment:
Why not testing, but I can't think about someone using it with more than 10 items.
Since performances is probably of shape `time = initialization + C * nb_of_items` better to keep a realistic number of items. Otherwise we are overweight the impact of the constant C

```suggestion
  const dataLength = 10;
```